### ISSUE TITLE
sequencing depth normalization 

### ISSUE THREAD

**Issue by Fougere87** [satijalab/seurat#422]
Hello, 
I was reading your paper about cortical inhibitory interneurons  https://www.nature.com/articles/nature25999 and I was wondering how you performed your normalization on sequencing depth of each cell using the theta overdispersion parameter. 
Will this feature be integrated in Seurat ? I would be interested to test it on our data but I'm not sure how to get the theta parameter.  
Thanks a lot !
Best regards

**Comment by Fougere87** [comment]
Hi, 
Already read this thread, but the function RegressOutNBreg is masked in the package (v2.3) I don't know why... And it's indicated [here](https://github.com/satijalab/seurat/blob/cfc2f29b448cd92204099264d3b6cf9737a82e85/R/preprocessing_internal.R) that it also corrects for cell cycle which I don't want to do for now...

**Comment by satijalab** [comment]
Thanks Leon. We  think this is a powerful method for preprocessing but is still under development (and quite slow), so we haven't integrated it into our public workflows. Hopefully Leon's answer solves the technical question you had.

**Comment by leonfodoulian** [comment]
Hi Christoph,

I can't find the commit you mentioned. 

Best,
Leon

**Comment by ChristophH** [comment]
Sorry, I did not merge the changes into the public repo. You should see the changes now.

**Comment by leonfodoulian** [comment]
Hi Christoph,

Could you please also export the function from Seurat in the develop branch? 

Thank you in advance!

Best,
Leon

**Comment by nfortelny** [comment]
Hi there,

I noticed in the version 2.0.1, FindMarkers was automatically run with latent.vars="nUMI", but this has been removed in the current version (now latent.vars = NULL). 

Thus when I now use NegBinDETest, the nUMI are not considered as latent variables any more by default. But this used to be the case (e.g. in version 2.0.1) - and made sense to me - so my question: why was it changed?

Thank you!
